source/material_model/rheology/visco_plastic.cc

```diff
@@ -256,12 +257,17 @@ namespace aspect
             // Step 3b: calculate current (viscous or viscous + elastic) stress magnitude
             double current_stress = 2. * viscosity_pre_yield * current_edot_ii;
 
-            // Step 4: calculate strain-weakened friction, cohesion
+            // Step 4a: calculate strain-weakened friction, cohesion
             const DruckerPragerParameters drucker_prager_parameters = drucker_prager_plasticity.compute_drucker_prager_parameters(j,
                                                                       phase_function_values,
                                                                       n_phases_per_composition);
             const double current_cohesion = drucker_prager_parameters.cohesion * weakening_factors[0];
-            const double current_friction = drucker_prager_parameters.angle_internal_friction * weakening_factors[1];
+            output_parameters.current_friction_angles[j] = drucker_prager_parameters.angle_internal_friction * weakening_factors[1];
```

Comment:
```suggestion
            output_parameters.current_friction_angles[j] = current_friction;
```

Reply:
Can you explain why? there is no current_friction variable at this point in the code...

Reply:
Ah I guess this is explained by your other comment below

Reply:
I might have been confused. Originally current_friction was computed here, but I see now that you have removed that line and just fill output_parameters.current_friction_angles. If you don't keep current_friction, than you can leave it as it is, also below! Not necessary to have two parameters with the same value around.

Reply:
ok ;) I changed it back in the new commit. Should we have two parameters (current_friction and output_parameters.current_friction_angles) to make it more intuitive or just one parameter (output_parameters.current_friction_angles) to not have the same value stored twice?